test/test_app_status.rb

```diff
@@ -4,6 +4,7 @@
 
 require "puma/app/status"
 require "rack"
+require "rack/mock"
```

Comment:
required but not used?

Reply:
I think this eliminated a 'circular require considered harmful' warning on [Ruby 2.2](https://github.com/puma/puma/runs/628616697).